src/lib/Liblog/setup_env.c

```diff
@@ -148,9 +148,6 @@ setup_env(char *filen)
 	} else if (errno != ENOENT) {
 		goto err;
 	}
-#ifdef WIN32
-	update_env_avltree();
-#endif
```

Comment:
why is it not required any more?

Reply:
Short version: Yes, as it is a dead code.

Long version: back in PBS v12, we had win_getenv() and win_putenv() (a wrapper of getenv() and putenv() in windows). And those were the only functions who were calling create_env_avltree(), and these funcs (update_env_avltree() and delete_env_avltree() make sense when you have called create_env_avltree(), aka global variable `env_avltree` in Libwin, is != NULL, if env_avltree is NULL then these funcs are like noop) so I removed it as they were using avl calls and this funcs are now dead code as no one is calling create_env_avltree() and env_avltree will be always NULL). And in PBS v13 we removed win_putenv and win_getenv, when we fixed one bug in multiple instances of mom on a single host using old windows installer.